.changeset/wise-toes-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@penumbra-zone/ui': major
+---
```

Comment:
Ideally we can:
- rename our old package to `@penumbra-zone/ui-deprecated` and set it to private
- name the new package `@penumbra-zone/ui`, set it as the old version `12.4.0`, and keep this major bump for that. After changeset versions everything, it should publish the new stuff.

However, this probably requires changing all the import paths. Maybe good for a next PR?

Reply:
Did it here: https://github.com/penumbra-zone/web/pull/1902